content-src/components/TopSites/_TopSites.scss

```diff
@@ -110,6 +110,10 @@ $hover-transition-duration: 150ms;
   padding: 0 $half-base-gutter;
   display: inline-block;
 
+  .top-site-icon.rich-icon {
+    border-radius: 4px;
```

Comment:
Any reason why we are changing corder-radius here and not in variable $top-sites-border-radius used in line 137 below

Reply:
$top-sites-border-radius is 6px